akka-slf4j/src/main/scala/akka/event/slf4j/Slf4jLogger.scala

```diff
@@ -107,12 +107,23 @@ class Slf4jLogger extends Actor with SLF4JLogging with RequiresMessageQueue[Logg
 
   @inline
   final def withMdc(logSource: String, logEvent: LogEvent)(logStatement: => Unit): Unit = {
+    logEvent match {
+      case m: LogEventWithMarker =>
+        val properties = m.marker.properties
+        if (properties.nonEmpty) {
+          properties.foreach { case (k, v) => MDC.put(k, String.valueOf(v)) }
```

Comment:
so they end up in MDC but users don't have to use an MDC like interface :+1:

Reply:
hmm so we `toString` them eagerly, even though they might be filtered out later. No way around it with the MDC API I guess, so :+1:

Reply:
Filter is typically in LoggingAdapter before the event reaches the Slf4jLogger, so the toString is rather lazy. MDC requires strings.